packages/botonic-core/src/models/knowledge-bases.ts

```diff
@@ -0,0 +1,7 @@
+export interface KnowledgeBaseResponse {
```

Comment:
I would name the file `knowledge-base.ts` to match `KnowledgeBaseResponse`

Reply:
but they are the types of the plugin-knowledge-bases.
Do I rename it to plugin-knowledge-bases?
Do I do the same with the files ai-agents and hubtype-analytics?

Reply:
so then just `KnowledgeBasesResponse`

Reply:
renamed KnowledgeBaseResponse to KnowledgeBasesResponse